index.html

```diff
@@ -19,16 +19,19 @@ <h2 class="subhead">New Activity</h2>
           <article class="categories">
             <h3 class="form-label">Select a category:</h3>
             <section class="category-box-wrapper">
-              <div class="category-boxes">
-                <img class="category-image" src="./assets/study.svg" alt="light bulb above an open book">
+              <div class="category-boxes study-box">
+                <img class="category-image study-image" src="./assets/study.svg" alt="light bulb above an open book">
+                <img class="category-image study-image-active hidden" src="./assets/study-active.svg" alt="green light bulb above an open book">
```

Comment:
Considering adding alt text = light bulb above an open book selected by user.

Reply:
yep, let's put this on the list